lib/galaxy/webapps/galaxy/buildapp.py

```diff
@@ -105,10 +105,11 @@ def paste_app_factory(global_conf, **kwargs):
 
     webapp.add_client_route('/admin/users', 'admin')
     webapp.add_client_route('/admin/roles', 'admin')
+    webapp.add_client_route('/admin/forms', 'admin')
     webapp.add_client_route('/admin/groups', 'admin')
     webapp.add_client_route('/admin/tool_versions', 'admin')
     webapp.add_client_route('/admin/quotas', 'admin')
-    webapp.add_client_route('/admin/forms/{form_id}', 'admin')
+    webapp.add_client_route('/admin/form/{form_id}', 'admin')
```

Comment:
This PR is branched from #4480. I updated the description above. We are changing this here to keep it consistent with the actual `forms` resource.